Packs/Axonius/Integrations/Axonius/axonius.yml

```
@@ -0,0 +1,102 @@
+name: Axonius
+display: Axonius
+description: This is the Axonius integration
+category: Data Enrichment & Threat Intelligence
+tests:
+- No tests
+commonfields:
+  id: Axonius
+  version: -1
+configuration:
+- name: ax_url
+  display: Server URL (e.g. https://example.net)
+  required: true
+  type: 0
+- name: ax_key
+  display: Axonius API Key
+  required: true
+  type: 0
+- name: ax_secret
+  display: Axonius API Secret
+  required: true
+  type: 4
+- name: insecure
+  display: Trust any certificate (not secure)
+  required: false
+  type: 8
```

I've seen you supported in proxy in the main function, if so please add a parameter for the integration to make it work.
```suggestion
- name: insecure
  display: Trust any certificate (not secure)
  required: false
  type: 8
- display: Use system proxy settings
  name: proxy
  required: false
  type: 8
```